src/handle-data.js

```diff
@@ -15,18 +25,8 @@ const MAX_CELLS = 10000;
  */
 export function getTotalInfo(isDim, layout, colIndex, numDims) {
   if (!isDim) return layout.qHyperCube.qGrandTotalRow[colIndex - numDims]?.qText;
-  if (colIndex !== 0) return '';
-  return layout.totals.label;
-}
-
-export function getHighestPossibleRpp(width, rowsPerPageOptions) {
-  const highestPossibleOption = [...rowsPerPageOptions].reverse().find((opt) => opt * width <= MAX_CELLS);
-  return highestPossibleOption || Math.floor(MAX_CELLS / width); // covering corner case of lowest option being too high
-}
-
-export function getColumnOrder({ qColumnOrder, qDimensionInfo, qMeasureInfo }) {
-  const columnsLength = qDimensionInfo.length + qMeasureInfo.length;
-  return qColumnOrder?.length === columnsLength ? qColumnOrder : [...Array(columnsLength).keys()];
+  if (colIndex === 0 && getColumnOrder(layout.qHyperCube)[0] === 0) return layout.totals.label;
```

Comment:
Running `getColumnOrder` here isn't the best idea. `getColumnInfo` runs for every column, which means you could end up running `getColumnOrder` 2000 times at most. Better to pass it through in this case, since it is already calculated before `getColumnInfo` runs